test/message_client/message_client.xml

```diff
@@ -1,9 +1,9 @@
 <config>
-    <component name="log_server" file="libcomponent_linux_log_server.so"/>
-    <component name="message_client" file="libcomponent_message_client.so">
+    <component name="log_server" implementation="linux_log_server"/>
+    <component name="message_client" implementation="message_client">
```

Comment:
Reading the specs, I wonder whether we should use `name` as a default for `implementation`? This could avoid a lot of redundancy.

Reply:
So `implementation` would be optional in case the name differs?